ux/musicplayer.java

```diff
@@ -4,6 +4,9 @@
 
 	What kind of methods does it need?
 	- play_music_file
-	- I don't know what else. How will it change based on user input?
```

Comment:
As in comment above.  The music will be categorized by level of intensity, which will be keyed to the greenhouse gas calculation.  (I might even have two pieces of music play at once if it is too "nice").